Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
     "ecpairing",
     "ecrecover",
     "ed25519",
+    "expmod",
```

Comment:
While the YP calls this expmod, every single client implementation (and the tests, including [geth](https://github.com/ethereum/go-ethereum/blob/master/core/vm/contracts.go#L149)) call it modexp.

Can we name it modexp?

Reply:
we can :smile:

Reply:
and we did